Tensile/TensileCreateLibrary.py

```diff
@@ -85,7 +80,7 @@ def processKernelSource(kernel, kernelWriterSource, kernelWriterAssembly):
     Returns (error, source, header, kernelName).
     """
     try:
-        kernelWriter = (
+        kernelWriter: KernelWriterAssembly | KernelWriterSource = (
```

Comment:
Can't use `|`...

Reply:
Removed, this type hint isn't very valuable anyways.